networkx/algorithms/lowest_common_ancestors.py

```diff
@@ -10,9 +10,125 @@
     "all_pairs_lowest_common_ancestor",
     "tree_all_pairs_lowest_common_ancestor",
     "lowest_common_ancestor",
+    "naive_lowest_common_ancestor",
+    "naive_all_pairs_lowest_common_ancestor",
 ]
 
 
+def get_a_lowest_common_ancestor(G, common_ancestors):
```

Comment:
```suggestion
def _get_a_lowest_common_ancestor(G, common_ancestors):
```

Just a subjective little nit pick: If the function isn't supposed to be "exported" outside the current module and if we don't want other people using an internal function we should start the function with `_`.